actions.go

```diff
@@ -46,18 +59,51 @@ type Action struct {
 	Information string
 	// The sub actions in the hierarchy. They do not inherit the permission requirement
 	SubActions map[string]*Action
-	// The function to execute when the action is called
-	Execute func(s *discordgo.Session, m *discordgo.MessageCreate, parameters []string) (string, error)
+	// Executed on call to format data passed to template
```

Comment:
Can the data structure contain functions ? Does the structure of the data impose restrictions ? Do we consider it as being bonded to a specific template ?
It would be great to include more information in the comment as it is one of the core element when defining an action.

Reply:
this is tied to what a `map[string]interface{}` is. there is nice explanation of that here https://bitfieldconsulting.com/golang/map-string-interface

In my opinion it is not worth adding explanation since it is part of golang itself